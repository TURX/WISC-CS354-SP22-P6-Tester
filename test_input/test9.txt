0 100 15