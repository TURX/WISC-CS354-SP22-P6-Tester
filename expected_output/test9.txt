Reading character starting at index 0 to 100 with an interval of 15

Reading character at index 0   : data = A : hit count = 0   : miss count = 1   : read data count = 1
Reading character at index 15  : data = r : hit count = 1   : miss count = 1   : read data count = 1
Reading character at index 30  : data = n : hit count = 2   : miss count = 1   : read data count = 1
Reading character at index 45  : data =   : hit count = 2   : miss count = 2   : read data count = 2
Reading character at index 60  : data = V : hit count = 3   : miss count = 2   : read data count = 2
Reading character at index 75  : data = D : hit count = 3   : miss count = 3   : read data count = 3
Reading character at index 90  : data =   : hit count = 4   : miss count = 3   : read data count = 3
